src/main/resources/files/liberty/server.xml.tpl

```diff
@@ -3,6 +3,9 @@
 
     <featureManager>
         <feature>microProfile-[# th:text="${mp_version}"/]</feature>
+        [# th:if="${mp_graphql}"]
+        <feature>mpGraphQL-1.0</feature>
+        [/]
     </featureManager>
 
```

Comment:
Could we also add this line:
```
<variable name="io.openliberty.enableGraphQLUI" value="true" />
```

This will enable the GraphiQL user interface which will make it easier for users to quickly write and execute queries/mutations.  It should be inserted at the same XML level as `<featureManager>` and `<httpEndpoint>`.